### ISSUE TITLE
Problem with SSH and putty

### ISSUE THREAD

**Issue by rsamboceu** [EliasKotlyar/Xiaomi-Dafang-Hacks#836]
I have the xiaomi dafang camera. I have loaded the cfw and also able to boot from the sdcard. Able to view and use the web gui. But for some reason I am not able to ssh into the camera because i wanted to see how much memory my model has. I am using putty and the ip address that is on the camera but saying that the location doesn't exist. But i am able to log into the web gui of the camera. 

I have not seen any post related to troubling using ssh. So I am submitting this as an issue.

**Comment by jmtatsch** [comment]
See that already shows you haven’t even read/understood/filled out the issue template.

**Comment by rsamboceu** [comment]
I saw many post that did not follow the issue template. So I guess you are just picking on me. Several post just state their problems without any information the template asked for. So I guess this community doesn't like to help. Though I like that this person has shared the hack with others and i am grateful because this is something really good and affordable. But on a case by case issue. People pick and choose who to help. Cause honestly I wouldn't ask any questions if I didnt think it was necessary. But other post are left opened and being supported though the template was not followed. thank you for your time. This is one project I can't continue on cause the lack of help. and discrimination against newbies.

**Comment by jmtatsch** [comment]
People help when they see you have tried something, at least sticked to the issue template rules. You have posted some generic questions on the chat and seemingly user caused ssh issue and expect other people to do the working for you. I would have helped, if you had adhered to the issue template but you were just to lazy. Now you are even getting snipish.

**Comment by rsamboceu** [comment]
I was not expecting people to do the work for me but at least guide me or give me some insight. The chat question was pretty generic i'll give you that but it was asked to see if anyone know the answer on the top of their head and could chime in. Or at least point me where i could find the information because i could not find anything in the issues list or google about it. and calling me insolent wasn't a nice thing at all. You help people that didnt follow the template but I actually provided more info than some people but yet you said you would of helped and decided not to help. I went to the chat as the template had said I tried to voice my questions before getting into the issue list. But anyways thanks for not helping me.
